Kiss.Bff/Program.cs

```diff
@@ -42,13 +42,20 @@
         Log.Fatal("Environment variable {variableKey} is missing", AuthorityKey);
     }
 
-    builder.Services.AddKissAuth(
-        authority,
-        builder.Configuration["OIDC_CLIENT_ID"],
-        builder.Configuration["OIDC_CLIENT_SECRET"],
-        builder.Configuration["OIDC_KLANTCONTACTMEDEWERKER_ROLE"],
-        builder.Configuration["OIDC_REDACTEUR_ROLE"]
-    );
+    builder.Services.AddKissAuth(options => 
+    {
+        options.Authority = authority;
+        options.ClientId = builder.Configuration["OIDC_CLIENT_ID"];
+        options.ClientSecret = builder.Configuration["OIDC_CLIENT_SECRET"];
+        options.KlantcontactmedewerkerRole = builder.Configuration["OIDC_KLANTCONTACTMEDEWERKER_ROLE"];
+        options.RedacteurRole = builder.Configuration["OIDC_REDACTEUR_ROLE"];
+        options.MedewerkerIdentificatieClaimType = builder.Configuration["OIDC_MEDEWERKER_IDENTIFICATIE_CLAIM"];
+        if(int.TryParse(builder.Configuration["OIDC_MEDEWERKER_IDENTIFICATIE_TRUNCATE"], out var truncate))
+        {
```

Comment:
default op 24 zetten, zodat het conform de huidige zgw standaard is?
in de readme een beetje uitleggen is wel fijn voor toekomstige devs. waarom is deze setting er. geldt ook voor OIDC_MEDEWERKER_IDENTIFICATIE_CLAIM. met eventueel een suggestie  als je met een azure AD werkt.
hoe zorgen we dat dit ook in de installatie handleiding komt. is daar ook een story voor?

Reply:
- default op 24: Daar twijfel ik over, omdat we dan een waarde moeten verzinnen waarmee je het uit zet. heb je een suggestie?
- readme: goed punt, ga ik naar kijken
- handleiding: dat hoort denk ik in deze story. ik zal een pr openen in de .github repository.

Reply:
je hebt gelijk over die default. laat die maar zitten

Reply:
Ik heb wat toegevoegd aan de readme, in de handleiding kunnen we het misschien wat uitgebreider toelichten

Reply:
dit lijkt me voldoende voor de readme. ongeveer hetzelfde moet in de installatie handleiding komen. om dat te borgen heb ik dat in de documentatie story vermeld https://dimpact.atlassian.net/jira/software/c/projects/PC/boards/259/backlog?selectedIssue=PC-160

Reply:
ps heb je ze ook al in de stork release ingesteld